### ISSUE TITLE
Assets being dumped into App folder

### ISSUE THREAD

**Issue by GuVAnj8Gv3RJ** [GuVAnj8Gv3RJ/NeosAccountDownloader#225]
Reported by @ThomFox and others on Discord.

We'll take a look!

**Comment by Lexevolution** [comment]
I got quite a few too when I downloaded my account contents today.
![image](https://github.com/GuVAnj8Gv3RJ/NeosAccountDownloader/assets/31176843/41151088-0e70-41ff-a13a-b12603fb915a)

In the mantime, should it be fine to move these to the Assets folder that was created?

**Comment by GuVAnj8Gv3RJ** [comment]
Yup feel free to move them back into your saved assets folder from the regular download.
